index.html

```diff
@@ -1,247 +1,300 @@
 <html>
-        <link rel="icon" href="tools-solid.png">
+<link rel="icon" href="tools-solid.png">
+
 <head>
-       <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-163874009-1"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'UA-163874009-1');
-</script>
-<!--Just Noble Mittal Things-->
-    <title>
-        Noble Mittal Blog
-    </title>
-    <link href="index.css" rel="stylesheet">
-    <meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-    <link href="https://fonts.googleapis.com/css2?family=Baloo+Tamma+2:wght@500&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet"> 
-        <style type="text/css">
-            .design{
-        width: auto;
-        height:auto;
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-163874009-1"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+
+    gtag('config', 'UA-163874009-1');
+  </script>
+  <!--Just Noble Mittal Things-->
+  <title>
+    Noble Mittal Blog
+  </title>
+  <link href="index.css" rel="stylesheet">
+  <meta charset=utf-8>
+  <meta http-equiv=X-UA-Compatible content="IE=edge">
+  <meta name=viewport content="width=device-width,initial-scale=1">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css2?family=Baloo+Tamma+2:wght@500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+  <style type="text/css">
+    .design {
+      width: auto;
+      height: auto;
+      border: 2px solid black;
+      border-radius: 5px;
+      background-color: #f4f4f4;
+      cursor: pointer;
+    }
+
+    @media screen and (max-width: 600px) {
+      .design {
+        width: 200px;
+        /* height: 100px; */
         border: 2px solid black;
         border-radius: 5px;
-background-color: #f4f4f4;
-cursor: pointer;
+        background-color: #f4f4f4;
+        cursor: pointer;
       }
-      .design:hover{
-       background-color: #555;
-        };
-      .design:click{
-        background-color: #444;
+
+      tr {
```

Comment:
Here, `tr` can be replaced by `.parent tr` as we want to change the styling inside the `parent div` container.
```suggestion
      .parent tr{
```